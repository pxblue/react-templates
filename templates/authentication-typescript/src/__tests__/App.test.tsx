import React from 'react';
import { render, screen } from '@testing-library/react';
import { test } from 'vitest';

test('renders welcome text', () => {
    render((<div>Welcome!</div>));
    expect(screen.getByText(/Welcome!/i)).toMatchSnapshot();
});
