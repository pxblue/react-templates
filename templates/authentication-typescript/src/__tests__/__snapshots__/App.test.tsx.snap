// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders welcome text 1`] = `
<div>
  Welcome!
</div>
`;
